.github/workflows/typos.yml

```diff
@@ -0,0 +1,25 @@
+name: "Typos"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches:
+      - master
+  schedule:
+    - cron: '45 9 * * 1'
```

Comment:
Why does this run on a schedule? What happens if it finds a spelling mistake during a scheduled run, how will it notify us?

I don't think it should run on pushes to master either. This will only cause annoying CI failures on master that nobody will fix anyway. This should just run on PRs where the PR author will fix it.